src/templates/app.vue

```diff
@@ -132,10 +133,17 @@ limitations under the License.
 
                     if (this.$data.data === text) this.$data.data = ""
 
-                    const domainLookup = await whoisJS(text)
+                    const domainLookup = await cfDNS(text, "NULL")
                     const json = await domainLookup.json()
-                    if (!json.domain) return this.error("Invalid domain.")
-                    this.$data.registrar = json.registrar.url
+                    if (json.Status !== 0) return this.error("Invalid domain.");
+                    (async() => {
```

Comment:
Would it not be cleaner to make this its own async function and then just call it whilst ignoring the returned promise?